### PR TITLE
USHIFT-620: Add a no-build option to configure-vm.sh script

### PR DESCRIPTION
**Pull request by ggiguash**
Closes [USHIFT-620](https://issues.redhat.com//browse/USHIFT-620)

### CODE REVIEW

File: scripts/devenv-builder/configure-vm.sh [hunk 1]
```diff
@@ -63,7 +76,9 @@ sudo subscription-manager config --rhsm.manage_repos=1
 sudo subscription-manager repos \
     --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
 #    --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
-sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
+if $BUILD_AND_INSTALL ; then
+    sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
+fi
 
 sudo cp -f ${OCP_PULL_SECRET} /etc/crio/openshift-pull-secret
```
Comment (jogeo) [inline scripts/devenv-builder/configure-vm.sh:83]: Copy of the pull secret should be conditional on BUILD_AND_INSTALL as well. /etc/crio does not necessarily  exist on a build system and copy of the pull secret into the ISO is already handled by the build.sh script.

File: scripts/devenv-builder/configure-vm.sh [hunk 2]
```diff
@@ -80,17 +95,20 @@ sudo firewall-cmd --permanent --zone=trusted --add-source=10.42.0.0/16
 sudo firewall-cmd --permanent --zone=trusted --add-source=169.254.169.1
 sudo firewall-cmd --reload
 
-# Run MicroShift Executable > Configuring MicroShift
```
Comment (jogeo) [inline scripts/devenv-builder/configure-vm.sh:83]: The modification of firewall rules should not be performed on a build host and should also be conditional on BUILD_AND_INSTALL.

**Comment by ggiguash** [comment]
@jogeo , can you review and approve the changes before we merge them?

**Review by jogeo — COMMENTED** [review]
Thanks for the latest updates. I've tested configure-vm.sh on the QE build host and things look good.

**Pull request merged**